apis/cluster/v1beta1/membercluster_types.go

```diff
@@ -50,6 +50,7 @@ type MemberClusterSpec struct {
 	HeartbeatPeriodSeconds int32 `json:"heartbeatPeriodSeconds,omitempty"`
 
 	// If specified, the MemberCluster's taints.
```

Comment:
do you want to add a comment mention that we don't allow update/delete taints and only adding is allowed?

Reply:
no delete after a taint is set on the cluster? I don't recall k8s has this behavior.

Reply:
I synced with Zhiying offline this comment was for tolerations and I've addresed it